README.md

```diff
@@ -39,78 +39,95 @@
 </p>
 </br>
 
-Semgrep is a fast, open-source, static analysis engine for finding bugs, detecting vulnerabilities in third-party dependencies, and enforcing code standards. Semgrep analyzes code locally on your computer or in your build environment: **code is never uploaded**. [Get started →.](#getting-started-)
+Semgrep is a fast static analysis engine designed to detect vulnerabilities, find bugs, and enforce code standards. Semgrep analyzes code locally on your computer or in your build environment: **by default, code is never uploaded**. [Get started →.](#getting-started-)
```

Comment:
Should we add another sentence here to communicate Semgrep does SAST, Supply Chain and Secrets scanning?

Reply:
Yes my suggestion would be:
```suggestion
Semgrep is an engine for scanning:
* **Code** for bugs & vulnerabilities using custom or pre-built rules
* **Supply Chain** dependencies for known vulnerabilities
* **Secrets** and hard-coded credentials that shouldn't be checked into source code
Semgrep analyzes code locally on your computer or in your build environment: **by default, code is never uploaded**. [Get started →.](#getting-started-)
```